### PR TITLE
Fix to facilitate old LinkedIn URL

### PR DESCRIPTION
**Pull request by fadils**
Some people still have their LinkedIn URL in the old fashion.
That is, instead of having a username-based URL i.e `http://linkedin.com/in/username`, their url is still `http://linkedin.com/pub/firstname-lastname/some-random-character/etc`

For example: `https://www.linkedin.com/pub/greg-street/7/644/b35` or 
my old linkedin url before I modify it: `https://www.linkedin.com/pub/fadil-sutomo/10/8aa/a5b`

In summary, the current format of simply supplying LinkedIn username might not work for some people

### CODE REVIEW

**Comment by barryclark** [comment]
Thanks for the pull request @fadils, much appreciated.

This is pretty opinionated, but I think that anyone including a link to their LinkedIn probably uses the new format... and if they don't, they really should, as it's a lot nicer. I noticed you updated yours to the new format too, @fadils :)

Going to keep as it, but thanks for bringing this up.

**Comment by fadils** [comment]
I did change mine. =)

Thanks for the good work. Keep it up.

Best Regards,
Fadil Sutomo
